package.json

```diff
@@ -64,6 +64,7 @@
     "moment": "2.10.6",
     "moment-timezone": "^0.4.0",
     "morgan": "1.2.0",
+    "ms": "0.7.1",
```

Comment:
I think the `ms` approach is nicer and cleaner.
```js
import ms from 'ms';
doByTime( ms( '2 days' ) );
doByTime( ms( '2 hours' ) );
```
vs
```js
import { DAY_IN_MS, HOUR_IN_MS } from 'time-constants';
doByTime( 2 * DAY_IN_MS );
doByTime( 2 * HOUR_IN_MS );
```
I suppose we pay for it with a little overhead in parsing text, but for most operations that won't really matter (and when it does, we can just use numbers). But I think it's overall cleaner, clearer, and aesthetically more pleasing. So count me on the side of "sure" for adding `ms`.